.github/workflows/integration-build-product.yml

```diff
@@ -1236,7 +1244,7 @@ jobs:
           TF_VAR_provision_ssh_pem: ${{ secrets.ssh_priv_key }}
           TF_VAR_ip_list: '["dhcp"]'
           TF_VAR_storage: "local-lvm"
-          TF_VAR_proxmox_host_list: '["mary"]'
+          TF_VAR_proxmox_host_list: '["${{ inputs.cluster }}"]'
           TF_VAR_vm_gateway: "10.10.1.254"
           TF_VAR_ip_mask: 23
           TF_VAR_bios: "ovmf"
```

Comment:
On line 1254, we could try to compare the differences of the image u22dci-gpu2 which is the one used by sanity tests and we know its on the 3 cluster nodes.
For this approach the image must exist on all.